server/memphis_handlers_poison_messages.go

```diff
@@ -461,11 +470,28 @@ cleanup:
 		} else if msgType == "schema" {
 			if _, value := idCheck[msgId]; !value {
 				idCheck[msgId] = true
+				filter := bson.M{"name": dlsMsg.Producer.Name, "station_id": station.ID, "connection_id": dlsMsg.Producer.ConnectionId}
+				var producer models.Producer
+				err = producersCollection.FindOne(context.TODO(), filter).Decode(&producer)
```

Comment:
I don't like the idea of getting the producer details in query time and not on insert